tests/fast_api_depends_test.py

```diff
@@ -5,7 +5,8 @@
 from websockets.exceptions import InvalidStatusCode
 
 # Add parent path to use local src as package for tests
-sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir)))
+sys.path.append(os.path.abspath(os.path.join(
```

Comment:
This seems inconsistent with the other changes here (preferring to have code in a single line)